assets/embedded-files/tekton/triggers.yaml

```diff
@@ -18,7 +18,7 @@ spec:
     - apiVersion: tekton.dev/v1beta1
       kind: PipelineRun
       metadata:
-        name: staging-pipeline-run-$(tt.params.appname)-$(uid)
+        name: staging-$(uid)
```

Comment:
maybe we should just use `uid`? After all we don't know how big the `uid` can be